src/test/java/uk/gov/companieshouse/overseasentitiesapi/validation/TrustDetailsValidatorTest.java

```diff
@@ -213,88 +199,196 @@ void testErrorReportedWhenCeaseDateFieldIsNotNullAndBosStillAssociatedWithTrust(
+        overseasEntitySubmissionDto.getBeneficialOwnersCorporate().get(0).setTrustIds(List.of());
+        overseasEntitySubmissionDto.getBeneficialOwnersIndividual().get(0).setTrustIds(List.of());
+        Errors errors = trustDetailsValidator.validate(overseasEntitySubmissionDto, new Errors(), LOGGING_CONTEXT, true);
+        String qualifiedFieldName = getQualifiedFieldName(OverseasEntitySubmissionDto.TRUST_DATA, TrustDataDto.IS_TRUST_INVOLVED_IN_OE);
+        String validationMessage = ValidationMessages.INVOLVED_IN_TRUST_WITHOUT_BENEFICIAL_OWNERS_ERROR_MESSAGE;
+        assertError(qualifiedFieldName, validationMessage, errors);
     }
 
     @Test
-    void testErrorReportedWhenCeaseDateIsBeforeCreationDate() {
-        disassociateBosFromTrust();
+    void testErrorReportedWhenTrustIsInvolvedInOverseasEntityAndNoBeneficialOwnersDueToCeasedDate() {
+        overseasEntitySubmissionDto.getBeneficialOwnersCorporate().get(0).setCeasedDate(LocalDate.of(1911,1, 1));
+        overseasEntitySubmissionDto.getBeneficialOwnersIndividual().get(0).setCeasedDate(LocalDate.of(1911,1, 1));
+        Errors errors = trustDetailsValidator.validate(overseasEntitySubmissionDto, new Errors(), LOGGING_CONTEXT, true);
+        String qualifiedFieldName = getQualifiedFieldName(OverseasEntitySubmissionDto.TRUST_DATA, TrustDataDto.IS_TRUST_INVOLVED_IN_OE);
+        String validationMessage = ValidationMessages.INVOLVED_IN_TRUST_WITHOUT_BENEFICIAL_OWNERS_ERROR_MESSAGE;
+        assertError(qualifiedFieldName, validationMessage, errors);
+    }
 
-        trustDataDtoList.get(0).setCeasedDate(LocalDate.of(1968,11, 30));
-        trustDataDtoList.get(0).setCreationDate(LocalDate.of(1972,3, 28));
 
+    @Test
+    void testErrorReportedWhenTrustIsInvolvedInOverseasEntityAndNoBeneficialOwnersDueToEmptyTrustNoc() {
+        overseasEntitySubmissionDto.getBeneficialOwnersCorporate().get(0).setTrusteesNatureOfControlTypes(List.of());
+        overseasEntitySubmissionDto.getBeneficialOwnersIndividual().get(0).setTrusteesNatureOfControlTypes(List.of());
         Errors errors = trustDetailsValidator.validate(overseasEntitySubmissionDto, new Errors(), LOGGING_CONTEXT, true);
+        String qualifiedFieldName = getQualifiedFieldName(OverseasEntitySubmissionDto.TRUST_DATA, TrustDataDto.IS_TRUST_INVOLVED_IN_OE);
+        String validationMessage = ValidationMessages.INVOLVED_IN_TRUST_WITHOUT_BENEFICIAL_OWNERS_ERROR_MESSAGE;
+        assertError(qualifiedFieldName, validationMessage, errors);
+    }
 
-        String qualifiedFieldName = getQualifiedFieldName(OverseasEntitySubmissionDto.TRUST_DATA, TrustDataDto.CEASED_DATE_FIELD);
-        String validationMessage = ValidationMessages.CEASED_DATE_BEFORE_CREATION_DATE_ERROR_MESSAGE.replace("%s", qualifiedFieldName);
+    @Test
+    void testErrorReportedWhenTrustIsInvolvedInOverseasEntityAndNoBeneficialOwnersDueNullTrustNoc() {
```

Comment:
I would change name to Due`To`NullTrustNoc to match naming in test method above.

Reply:
Added the "to"